src/ChannelCallback.h

```diff
@@ -0,0 +1,12 @@
+#include "Bindings/PluginLua.h"
+
+class cChannelCallback
```

Comment:
You cannot make this a lambda, because it has multiple functions (`Call` and `BelongsTo`); a lambda could wrap the callback, but wouldn't be capable of saying whether it belongs to a certain plugin or not.

However, I do believe that this class could be made internal to the `cChannelManager` class, and moved in its source and header files.